lib/package.json

```diff
@@ -52,6 +52,11 @@
       "types": "./dist/types/src/nanoindex.d.ts",
       "require": "./dist/cjs/src/nanoindex.js",
       "import": "./dist/web/src/nanoindex.js"
+    },
+    "./platform": {
```

Comment:
TODO: verify if the name `platform` is expected here. This means  the consumers will use the import as so
```
from "@opentdf/sdk/platform"
```